src/day1/array_ext.rb

```diff
@@ -0,0 +1,25 @@
+#: Extend Array with scan_zzz methods (see method docs).
+
+class Array
```

Comment:
Although extending Array is possible, it's considered a bad practice for general code. It should be done only when the added method is used extensively (i.e. in Rails).

Reply:
Got it, thanks, fixed in 5c87212.